ideascube/mediacenter/views.py

```diff
@@ -47,9 +50,17 @@ def _set_available_kinds(self, context):
             (kind, label) for kind, label in Document.KIND_CHOICES
             if kind in available_kinds]
 
+    def _set_available_package_ids(self, context):
+        available_package_ids = self._search_for_attr_from_context('package_id', context)
+        catalog = catalog_mod.Catalog()
+
+        context['available_package_ids'] = {
+            package.id:package.name for package in catalog.list_installed(available_package_ids)}
+
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         self._set_available_kinds(context)
+        self._set_available_package_ids(context)
         self._set_available_langs(context)
         self._set_available_tags(context)
```

Comment:
I didn't know. That's nice.

But the available_* are compute from the context. Is it possible to get the context from the view (self in the property) ?